include/contractor/graph_contractor.hpp

```diff
@@ -401,7 +409,7 @@ class GraphContractor
     bool Bias(const NodeID a, const NodeID b) const;
 
     std::shared_ptr<ContractorGraph> contractor_graph;
-    stxxl::vector<QueryEdge> external_edge_list;
+    ExternalVector<QueryEdge> external_edge_list;
```

Comment:
Did you check if this increases the peak memory consumption for `osrm-contract`?